test_scripts/Smoke/API/008_CreateInteractionChoiceSet_PositiveCase_SUCCESS.lua

```diff
@@ -0,0 +1,103 @@
+---------------------------------------------------------------------------------------------------
+-- User story: Smoke
+-- Use case: CreateInteractionChoiceSet
+-- Item: Happy path
+--
+-- Requirement summary:
+-- [CreateInteractionChoiceSet] SUCCESS
+--
+-- Description:
+-- Mobile application sends valid CreateInteractionChoiceSet request with
+-- {interactionChoiceSetID, ChoiceSet: [(choiceID1, vrCommands, params),
+-- (choiceID2, vrCommands, params)] and SDL successfully stores UI-related
+-- choices and gets successful responses to corresponding VR.AddCommands
+-- (VR-related choices) from HMI.
+
+-- Pre-conditions:
+-- a. HMI and SDL are started
+-- b. appID is registered and on SDL
+-- c. appID is in Background, Full or Limited HMI level
+
+-- Steps:
+-- appID requests CreateInteractionChoiceSet with valid parameters
+
+-- Expected:
+-- SDL validates parameters of the request
+-- SDL checks if VR interface is available on HMI
+-- SDL checks if CreateInteractionChoiceSet is allowed by Policies
+-- SDL checks if all parameters are allowed by Policies
+-- SDL transfers the VR.AddCommand with allowed parameters to HMI
+-- SDL receives successful responses to corresponding VR.AddCommands from HMI
+-- SDL responds with (resultCode: SUCCESS, success:true) to mobile application
+---------------------------------------------------------------------------------------------------
+
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local commonSmoke = require('test_scripts/Smoke/commonSmoke')
+
+--[[ Local Variables ]]
+local putFileParams = {
+	requestParams = {
+	    syncFileName = 'icon.png',
+	    fileType = "GRAPHIC_PNG",
+	    persistentFile = false,
+	    systemFile = false
+	},
+	filePath = "files/icon.png"
+}
+
+local requestParams = {
+	interactionChoiceSetID = 1001,
+	choiceSet = {
+		{
+			choiceID = 1001,
+			menuName ="Choice1001",
+			vrCommands = {
+				"Choice1001"
+			},
+			image = {
+				value ="icon.png",
+				imageType ="DYNAMIC"
+			}
+		}
+	}
+}
+
+local responseVrParams = {
+	cmdID = requestParams.interactionChoiceSetID,
+	type = "Choice",
+	vrCommands = requestParams.vrCommands
+}
```

Comment:
Add grammarID

Reply:
@GetmanetsIrina done in https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/1716/commits/7822eee4408704d77053a8a725acc23c5f5b4ea6